docs/reference/docs/bulk.asciidoc

```diff
@@ -128,27 +128,21 @@ field. It automatically follows the behavior of the index / delete
 operation based on the `_parent` / `_routing` mapping.
 
 [float]
-[[bulk-consistency]]
-=== Write Consistency
+[[bulk-wait-for-active-shards]]
+=== Wait For Active Shards
 
 When making bulk calls, you can require a minimum number of active
-shards in the partition through the `consistency` parameter. The values
-allowed are `one`, `quorum`, and `all`. It defaults to the node level
-setting of `action.write_consistency`, which in turn defaults to
-`quorum`.
-
-For example, in a N shards with 2 replicas index, there will have to be
-at least 2 active shards within the relevant partition (`quorum`) for
-the operation to succeed. In a N shards with 1 replica scenario, there
-will need to be a single shard active (in this case, `one` and `quorum`
-are the same).
+shard copies before proceeding with the `wait_for_active_shards` 
```

Comment:
this reads to me like we're proceed with the `wait_for_active_shards` parameter. Maybe change to 

When making bulk calls, you can set the `wat_for_active_shards` parameter to require a minimum of shard copies to be active before starting to process the bulk request.